meetings/2017-12-13.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-12-13
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=1sHufGWR_WU
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/435
+* **Minutes Google Doc**: https://docs.google.com/document/d/1oWrjw845TDCAYCZHkYv51oyA_d-jxzcGe0hvwtf1_Io
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Michaël Zasso @targos (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Rich Trott @Trott (TSC)
+
+
+
+## Agenda
+
+### Announcements
+* Collab summit Berlin, most likely on May 31 - Friday, June 1, 2018. The 2 days prior to JSConf.eu [#56](https://github.com/nodejs/summit/issues/56).
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+
+Rich: GitHub conversation did not get “unstuck”. Move to vote.
```

Comment:
What was the outcome here? Was a vote taken? If not, when will it be taken?

Reply:
IIRC we want for the TSC members to vote in the issue. We might already have enough approvals there, let me check.

Reply:
Not enough votes yet. Left a comment.

Reply:
https://github.com/nodejs/node/pull/17156#issuecomment-351457062